tests/execution/test-cors-tailcall-run.md

```diff
@@ -0,0 +1,110 @@
+# Verify the correct CORS headers for requests from https://tailcall.run
+
+```graphql @config
+schema
+  @upstream(baseURL: "http://example.com")
+  @server(
+    port: 8000
+    headers: {
+      cors: {allowHeaders: ["*"], allowMethods: ["GET", "POST", "OPTIONS"], allowOrigins: ["https://tailcall.run"]}
```

Comment:
The issue is intended to test headers that put automatically for `tailcall.run` and therefore we don't need to specify headers in test - we need to test that it works without headers